spyder/widgets/ipythonconsole/shell.py

```diff
@@ -110,9 +114,15 @@ def get_cwd(self):
             return
         else:
             self.silent_exec_method(code)
+            
+    def set_bracket_matcher_color_scheme(self, color_scheme):
```

Comment:
Please add a docstring here. Something like `"""Set color scheme for matched parentheses."""`